<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>565</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>凯撒加密实用工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QCheckBox,QRadioButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  color: #454545;
}
QPlainTextEdit, QLineEdit
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#fff8e1;
  border:2px solid #ffcc80;
  border-radius:5px;
  color: #454545;
  padding: 6px 6px
}

QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(134,183,200);
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#4dd0e1;
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#00bcd4; 
  border:2px solid #0097a7;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,3">
   <item>
    <layout class="QHBoxLayout" name="up" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>密文：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="inText">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16000000</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="paste">
         <property name="text">
          <string>粘贴</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(248,187,208);
  border:2px solid #f06292;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color: #f48fb1;
  border:2px solid #ec407a;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#f44336; 
  border:2px solid #d81b60;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="down" stretch="0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rAuto">
           <property name="text">
            <string>自动破解</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rForce">
           <property name="text">
            <string>暴力穷举</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rManual">
           <property name="text">
            <string>手动模式</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>密钥</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="keyLine">
             <property name="maximumSize">
              <size>
               <width>601</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#c5e1a5;
  border:2px solid #9ccc65;
  border-radius:5px;
  color: #454545;
  padding: 6px 6px</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="plus">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#9ccc65;
  border:2px solid #7cb342;
  border-radius:5px;
  color: white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#7cb342;
  border:2px solid #689f38;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#558b2f; 
  border:2px solid #33691e;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="minus">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#9ccc65;
  border:2px solid #7cb342;
  border-radius:5px;
  color: white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#7cb342;
  border:2px solid #689f38;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#558b2f; 
  border:2px solid #33691e;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reverseKey">
             <property name="styleSheet">
              <string notr="true">QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#9ccc65;
  border:2px solid #7cb342;
  border-radius:5px;
  color: white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#7cb342;
  border:2px solid #689f38;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#558b2f; 
  border:2px solid #33691e;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
             </property>
             <property name="text">
              <string>反转密钥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="outText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="reverse">
         <property name="text">
          <string>反转</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copy">
         <property name="text">
          <string>复制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refresh">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#9ccc65;
  border:2px solid #7cb342;
  border-radius:5px;
  color: white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#7cb342;
  border:2px solid #689f38;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#558b2f; 
  border:2px solid #33691e;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
